public/js/src/buildit/buildit_plan.tag

```diff
@@ -229,8 +229,11 @@
                 break;
             }
         }
+        
+
+        console.log('foo', index)
```

Comment:
Is this intentional?  You want to log index to console.log everytime?  Looks like it might be left over from local testing.